rllib/tuned_examples/ppo/pendulum-ddppo.yaml

```diff
@@ -0,0 +1,20 @@
+pendulum-ddppo:
+    env: Pendulum-v1
+    run: DDPPO
+    stop:
+        episode_reward_mean: -300
```

Comment:
debugging?

Reply:
Which part?
* reward: It's able to get to -300.
* timesteps: Yeah, it does need up to 1M sometimes. DD-PPO is not a very stable algo, it seems. Especially on cont. action tasks. But even on Atari I have yet to find a good choice of hyperparams.

Reply:
Oh for some reason I read this as CartPole and not Pendulum, my b

Reply:
Haha, yeah -300 would be pretty bad for CartPole :)